.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: enhancement
```

Comment:
Maybe leave unlabeled or use a new "triage" label? It would make it easier to track new issues for triage? Same below.

Reply:
Yeah, let me create a `triage` label. I think we can still auto-bin based on issue category, but we should have a human pass to confirm.